PCampReview.py

```diff
@@ -158,15 +155,80 @@ def infoPopup(self,message):
     messageBox = qt.QMessageBox()
     messageBox.information(None, 'Slicer mpMRI review', message)
 
+  def setupIcons(self):
```

Comment:
I would like to know if there is a minimum resolution for using slicer? Is there any default icon size used for slicer tabbars?